use-timescale/integrations/power-bi.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Integrate Power BI with Timescale Cloud
+excerpt: Integrate Timescale Cloud with Power BI for advanced data visualization.
+products: [cloud, mst, self_hosted]
+keywords: [Power BI, visualizations, analysis, real-time]
+---
+
+import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.mdx";
+
+# Integrate Power BI with $CLOUD_LONG
+
+[Power BI][power-bi] is a business analytics tool for visualizing data, creating interactive reports, and sharing insights across an organization.
+
+This page explains how to integrate Power BI with $CLOUD_LONG using the PostgreSQL ODBC driver, so that you can build interactive reports based on the data in your $SERVICE_LONG. 
+
+## Prerequisites
+
+<IntegrationPrereqs />
+
+- Download [Power BI Desktop][power-bi-install] on your Microsoft Windows machine.
```

Comment:
Specifying Windows is redundant, it is only available for Windows.

Reply:
I'd prefer to shout it out for people such as I who clicked about a few times when i read the doc before I realized i can't install it on an *NIX machine.